nat-lab/tests/utils/connection_tracker.py

```diff
@@ -92,8 +102,10 @@ def __init__(
         self._connection: Connection = connection
         self._config: Optional[List[ConnectionTrackerConfig]] = configuration
         self._events: List[FiveTuple] = []
+        self._lock: Lock = threading.Lock()
 
         self._initialized: bool = False
+        self._synchronize: SynchronizeState = SynchronizeState.NOT_SYNCHRONIZED
         self._init_connection: FiveTuple = FiveTuple(
```

Comment:
Let's rename it to sync connection?
Also maybe we should ping somthing like 127.0.0.2, to make it more unique? ()

Reply:
Good idea